functorch/_src/eager_transforms.py

```diff
@@ -1305,6 +1305,20 @@ def get_function_def(sig):
     torch.jit._register_decomposition(decomp, graph)
 
 
+# use an alternate way to register an operator into the decomposition table
+# _register_jit_decomposition doesn't work for some operators, e.g. addr,
+#  because the Tensor types generated cannot be unioned by torchscript
+# decomp should be type OpOverload
+_register_jit_decomposition_lib = torch.library.Library("aten", "IMPL", "FuncTorchBatched")
```

Comment:
This isn't using the `_jit` path - I'd just call it something like `vmap_decompositions_lib`.